flake.nix

```diff
@@ -606,6 +594,20 @@
 
           hardeningDisable = [ "pie" ];
         };
+        dockerImage =
```

Comment:
Since this adds an attribute to the overlay, it may be better to use a more descriptive name like `nixDockerImage`.

Reply:
That shouldn’t since I’ve only added it to `packages` and `overlay` doesn’t include these.

But I can move it to `nix.passthru.dockerImage` (and then include it in the overlay) or something like that if you prefer